automation_py/automations/adsense_creator/code_site.py

```
@@ -0,0 +1,866 @@
+import logging
+import time
+import os
+import re
+import glob
```

Ruff (0.11.9)

`glob` imported but unused

Remove unused import: `glob`

(F401)